README.md

```diff
@@ -96,7 +81,48 @@ It is recommended that you start by cloning the implementation of one of the exi
 
  Current Example Implementations:
 
-* [Quectel BG96](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Library/tree/master/modules/bg96)
-* [Sierra Wireless HL7802](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Library/tree/main/modules/hl7802)
-* [U-Blox Sara-R4](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Library/tree/main/modules/sara_r4)
+* [Quectel BG96](https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface/tree/main/modules/bg96)
+* [Sierra Wireless HL7802](https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface/tree/main/modules/hl7802)
+* [U-Blox Sara-R4](https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface/tree/main/modules/sara_r4)
+
+## Building Unit Tests
+
+### Checkout CMock Submodule
+By default, the submodules in this repository are configured with `update=none` in [.gitmodules](.gitmodules) to avoid increasing clone time and disk space usage of other repositories (like [amazon-freertos](https://github.com/aws/amazon-freertos) that submodules this repository).
+
+
+To build unit tests, the submodule dependency of CMock is required. Use the following command to clone the submodule:
+```
+git submodule update --checkout --init --recursive --test/unit-test/CMock
+```
+
+### Platform Prerequisites
+
+- For building the library, **CMake 3.13.0** or later and a **C90 compiler**.
```

Comment:
"For building the library," this part should describe "running unit tests".

Reply:
This is referred from [device shadow doc](https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk#platform-prerequisites), and I think the correct description should be `For building the unit test` because the only CMakeLists.txt exist in test/CMakeLists.txt.